pyteal/compiler/sourcemap.py

```diff
@@ -937,7 +937,12 @@ def _build_pc_sourcemap(self):
         algod = algod_with_assertion(
             self.algod, msg="Adding PC's to sourcemap requires live Algod"
         )
-        algod_compilation = algod.compile(self.compiled_teal(), source_map=True)
+
+        teal: str = self.compiled_teal()
+        for placeholder in pt.Tmpl.session_templates():
+            teal = teal.replace(placeholder, pt.Tmpl.zero(placeholder))
+
+        algod_compilation = algod.compile(teal, source_map=True)
```

Comment:
minor question: so this change is to replace template vars with an algod compilable instance, such that we can have same PC, right?

Reply:
yep, same question as bob raised above.